src/libraries/System.Net.Http/tests/FunctionalTests/System.Net.Http.Functional.Tests.csproj

```diff
@@ -8,278 +8,176 @@
-             Link="Common\System\Net\Capability.Security.cs" />
-    <Compile Include="$(CommonTestPath)System\Net\Capability.Security.Windows.cs" Condition="'$(TargetsWindows)' == 'true'"
-             Link="Common\System\Net\Capability.Security.Windows.cs" />
-    <Compile Include="$(CommonTestPath)System\Net\Capability.Security.Unix.cs" Condition="'$(TargetsUnix)' == 'true' or '$(TargetsBrowser)' == 'true'"
-             Link="Common\System\Net\Capability.Security.Unix.cs" />
-    <Compile Include="$(CommonTestPath)System\Net\Configuration.cs"
-             Link="Common\System\Net\Configuration.cs" />
-    <Compile Include="$(CommonTestPath)System\Net\Configuration.Http.cs"
-             Link="Common\System\Net\Configuration.Http.cs" />
-    <Compile Include="$(CommonTestPath)System\Net\Configuration.Security.cs"
-             Link="Common\System\Net\Configuration.Security.cs" />
-    <Compile Include="$(CommonTestPath)System\Net\Configuration.Sockets.cs"
-             Link="Common\System\Net\Configuration.Sockets.cs" />
-    <Compile Include="$(CommonTestPath)System\Net\TestWebProxies.cs"
-             Link="Common\System\Net\TestWebProxies.cs" />
-    <Compile Include="$(CommonTestPath)System\Net\VerboseTestLogging.cs"
-             Link="Common\System\Net\VerboseTestLogging.cs" />
-    <Compile Include="$(CommonTestPath)System\Net\Http\LoopbackProxyServer.cs"
-             Link="Common\System\Net\Http\LoopbackProxyServer.cs" />
-    <Compile Include="$(CommonTestPath)System\Net\Http\LoopbackServer.cs"
-             Link="Common\System\Net\Http\LoopbackServer.cs" />
-    <Compile Include="$(CommonTestPath)System\Net\Http\LoopbackServer.AuthenticationHelpers.cs"
-             Link="Common\System\Net\Http\LoopbackServer.AuthenticationHelpers.cs" />
-    <Compile Include="$(CommonTestPath)System\Net\Http\GenericLoopbackServer.cs"
-             Link="Common\System\Net\Http\GenericLoopbackServer.cs" />
-    <Compile Include="$(CommonTestPath)System\Net\Http\HttpClientHandlerTestBase.cs"
-             Link="Common\System\Net\Http\HttpClientHandlerTestBase.cs" />
-    <Compile Include="$(CommonTestPath)System\Net\Http\TestHelper.cs"
-             Link="Common\System\Net\Http\TestHelper.cs" />
-    <Compile Include="$(CommonTestPath)System\Threading\TrackingSynchronizationContext.cs"
-             Link="Common\System\Threading\TrackingSynchronizationContext.cs" />
-    <Compile Include="$(CommonTestPath)System\Threading\Tasks\GetStateMachineData.cs"
-             Link="Common\System\Threading\Tasks\GetStateMachineData.cs" />
-    <Compile Include="$(CommonTestPath)System\Threading\Tasks\TaskTimeoutExtensions.cs"
-             Link="Common\System\Threading\Tasks\TaskTimeoutExtensions.cs" />
+    <Compile Include="$(CommonPath)Interop\Unix\Interop.Libraries.cs" Condition="'$(TargetsUnix)' == 'true' or '$(TargetsBrowser)' == 'true'" Link="Common\Interop\Unix\Interop.Libraries.cs" />
```

Comment:
Please revert the unrelated changes made by VS. I.e.: `Link` attribute should be on the next line.

Reply:
It's concerning that VS is doing this automatically. @safern do you know what is causing this to happen now? I don't want to revert this code with every new PR that alters project files :/

Reply:
IIRC, When you update something via VS or add a new file it trims the new lines. This was one of the downsides of moving the Link metadata declaration inline instead of using the extended XML tag format.

@davkean do you know how can we avoid this?

Reply:
I talked about it with @ViktorHofer and he suggested to report it in VS. So I did, here's the [ticket](https://developercommunity.visualstudio.com/content/problem/1052431/csproj-file-gets-reformatted-upon-save.html). Please upvote if you want some reaction. Although, I'm highly skeptical about it.